tools/autograd/derivatives.yaml

```diff
@@ -1020,6 +1020,9 @@
 - name: symeig(Tensor self, bool eigenvectors=False, bool upper=True) -> (Tensor eigenvalues, Tensor eigenvectors)
   self: symeig_backward(grads, self, eigenvectors, upper, eigenvalues, eigenvectors_return)
 
+- name: swapdim(Tensor(a) self, int dim1, int dim2) -> Tensor(a)
```

Comment:
You won't need to add a derivatives.yaml entry if this is implemented using the alias instructions.